Add analytics section describing user event data (#109)
Kong/insomnia-docs: docs/insomnia/analytics-collected.md

---
layout: article-detail
title:  User Analytics Collected
category: Security
category-url: security
---

> When you first download Insomnia, you were asked if you wanted to send analytics data to the Insomnia team about your Insomnia Apps performance and your behavior in the app.  For those who clicked "yes", this article is a description of which analytics are collected and sent.

Reviewer: should we add anything here about how they can check what their status is in the settings (and how to change/remove them in the settings)?

Author: Done

---
layout: article-detail
title:  User Analytics Collected
category: Security
category-url: security
---

> When you first download Insomnia, you were asked if you wanted to send analytics data to the Insomnia team about your Insomnia Apps performance and your behavior in the app.  For those who clicked "yes", this article is a description of which analytics are collected and sent.

Reviewer 2: ```suggestion
When you first download Insomnia, you were asked if you wanted to send analytics data to the Insomnia team about your Insomnia Apps performance and your behavior in the app. This article is a description of what analytics are collected and sent.
```

> You can edit your preference on sharing analytics data with Insomnia via the Insomnia App Preference Page by scrolling down to the "Network Activity" section and checking or un-checking the box next to "Send Usage Statistics"

Reviewer 2: ```suggestion
You can edit your preference on sharing analytics data with Insomnia via the Insomnia app Preference Page by scrolling down to the **Network Activity** section and checking or unchecking the box next to **Send Usage Statistics**
```

# Error Data Collection

> When you opt-in to collection of analytics, the Insomnia app will send anonymized action event data to Kong that may be later used to evaluate user behavior in the purpose of guiding product decisions.

Reviewer 2: ```suggestion
When you opt-in to the collection of analytics, the Insomnia app will send anonymized action event data to Kong that may later be used to evaluate user behavior for the purpose of guiding product decisions.
```

> Here is the format of the JSON data body of the event sent:

Reviewer 2: ```suggestion
This is the format of the JSON data body for a sent event:
```
